docs/delegates/8b000cc3de75b75d2d3e845770dbb13a5d3827d730098f3eb8d921f23b694c2e/README.md

```diff
@@ -0,0 +1,40 @@
+---
+unikid: 8b000cc3de75b75d2d3e845770dbb13a5d3827d730098f3eb8d921f23b694c2e
+twitter: bpresles
+github: bpresles
+forum: bpresles
+email: bertrand@presles.fr
+website: https://www.cloud-libre.eu
+
+---
+
+## Summary
+Hello everyone ! :hand:
+
+I'm a french :fr: web and mobile developer and architect based in Nantes and I'm interested by blockchain since 2017 and also more generally in decentralized technology for 2 years.
+
+I'm an active participant of the Duniter blockchain and user of the associated crypto-currency, the Ğ1. I also contribute, in my spare time, to the development of the wallet app of this crypto-currency, Cesium and I'm also the leader of the FideliĞ1 project which use the Ğ1 as token of fidelity programs for shops, restaurants, pubs, associations, ...
+
+In my professional life I design and develop mainly websites and hybrid mobile applications. I'd love to work on projects which use decentralized technologies in my professional work, but I didn't have the opportunity yet.
+
+In my personal life, I'm a supporter of the Nantes' hockey team, the Corsaires de Nantes, an amateur rock climber and a science lover.
+
+I've known the Unikname project first through the Blockchain & Société association and I'm a delegate since the beginning.
+
+As a delegate, I'm committed to maintain an up and running uns.network forging node (both livenet and sandbox networks)
+
+If you want to support me, open `delegate support` feature from your app then select `@bpresles`.
+
+## Contributions
+- **[since 2019]** Delegate on uns.network - https://delegates.uns.network/delegates/8b000cc3de75b75d2d3e845770dbb13a5d3827d730098f3eb8d921f23b694c2e/
+
+- **[since 2019]** Active member of the Blockchain & Société association - https://blockchainsociete.org/
+
+- **[since 2019]** Active member of the Ğ1 crypto-currency. Contributing developer of the Cesium wallet application - https://git.duniter.org/clients/cesium-grp/cesium
+
+- **[since 2021]** Co-founder of the FideliĞ1 project. Lead developer on this project
+
+    https://git.duniter.org/bpresles/fidelig1card
+
+    https://git.duniter.org/bpresles/fidelig1app
```

Comment:
These links aren't browseable. Is this intended?

Reply:
I've no issue to browse them. They are public and accessible to anyone.

Reply:
I fixed the links not being clickable.